src/components/Profile.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import '../styles/profile.css';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Table from 'react-bootstrap/Table';
+import Alert from 'react-bootstrap/Alert';
+import {
+  startLoading,
+  endLoading,
+  setMissions,
+} from '../redux/missionsSlice';
+import { getMissionsApi } from '../redux/missionApi';
+
+const Profile = () => {
+  const { rocketList } = useSelector((state) => state.rockets);
+  const reservedRockets = rocketList.filter((rocket) => rocket.reserved);
+  const { missionList, joinedIds } = useSelector((store) => store.mission);
```

Comment:
- [ Required ] I think it would be better if you use the `filter` method to fetch the missions a user has joined. A mission a user has joined ought to have the `reserved` key set to `true` making it possible for you to select the missions a given user has joined. You've done this for the reserved rockets please do the same for the missions.

## Example code
```JavaScript
   const { missionList } = useSelector((state) => state.missions);
   const joinedMissions = missionList.filter((rocket) => mission.reserved);
```